### PR TITLE
Anagram.java - Roobini

### PR DESCRIPTION
**Pull request by sroobini**
false
true
true
true
false

### CODE REVIEW

File: Anagram.java - Roobini [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+				count = 1;
+				myMap.put(input.charAt(i) , count);
+			}
+		}
+		
+		for(int i = 0; i < input1.length() ; i++){
+			if(myMap.containsKey(input1.charAt(i))){
+			myMap.put(input1.charAt(i) , count-1);
+		}
+			else{
```
Comment (chandramouly) [inline Anagram.java - Roobini:33]: Good attempt? O(n) solution..
Why this else block? I couldn't understand the significance?

File: Anagram.java - Roobini [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class Anagram {
+
+	public static boolean isAnagram(String input , String input1){
```
Comment (chandramouly) [inline Anagram.java - Roobini:7]: can we use better parameter name??
String input , String input1

File: Anagram.java - Roobini (hunk 1 above)
Comment (sroobini) [reply Anagram.java - Roobini:33]: This Else block is to reset the count to 1 when a new character comes which is not in the map.

File: Anagram.java - Roobini (hunk 2 above)
Comment (sroobini) [reply Anagram.java - Roobini:7]: okay 👍

**Pull request closed without merging**